tests/models.py

```diff
@@ -11,6 +11,9 @@ class BasicModel(DataModel):
     """
     schema_url = "http://example.com/schemas/basic_model"
 
+    def get_primary_array_name(self):
+        return "data"
+
```

Comment:
Isn't the default primary array name `data`?  Is this needed?

Reply:
Lemme try again without it, I thought it wasn't working without that but I could well have been confused.

Reply:
You're right, works fine without it, will remove.